RecoLocalMuon/RPCRecHit/src/CSCSegtoRPC.cc

```diff
@@ -55,8 +49,44 @@ ObjectMapCSC::ObjectMapCSC(const edm::EventSetup& iSetup){
     }
   }
 }
+
+void ObjectMapCSC::FillObjectMapCSC(const edm::EventSetup& iSetup){
+    edm::ESHandle<RPCGeometry> rpcGeo;
+    edm::ESHandle<CSCGeometry> cscGeo;
+    
+    iSetup.get<MuonGeometryRecord>().get(rpcGeo);
+    iSetup.get<MuonGeometryRecord>().get(cscGeo);
+    
+    for (TrackingGeometry::DetContainer::const_iterator it=rpcGeo->dets().begin();it<rpcGeo->dets().end();it++){
+        if(dynamic_cast< const RPCChamber* >( *it ) != 0 ){
+            auto ch = dynamic_cast< const RPCChamber* >( *it );
+            std::vector< const RPCRoll*> roles = (ch->rolls());
+            for(std::vector<const RPCRoll*>::const_iterator r = roles.begin();r != roles.end(); ++r){
+                RPCDetId rpcId = (*r)->id();
+                int region=rpcId.region();
+                if(region!=0){
+                    int station=rpcId.station();
+                    int ring=rpcId.ring();
+                    int cscring=ring;
+                    int cscstation=station;
+                    RPCGeomServ rpcsrv(rpcId);
+                    int rpcsegment = rpcsrv.segment();
+                    int cscchamber = rpcsegment; //FIX THIS ACCORDING TO RPCGeomServ::segment()Definition
```

Comment:
I have no idea, my purpose was only to migrate it to multithreading. Including @pietverwilligen , for the RPC DPG